_includes/post_info.html

```diff
@@ -7,9 +7,7 @@
       <img src="{{ author.avatar | relative_url }}">
     {% endif %}
     <p class="meta">
-      {% if author.name %}{{ author.name }} - {% endif %}
-      {% assign x = date | date: "%m" | minus: 1 %}
-      {{ site.data.language.str_months[x] }} {{ date | date: "%d, %Y" }}
+    {% if author.name %}{{ author.name }} - {% endif %} - {% endif %}{{ date | date: "%B %-d, %Y" }}
```

Comment:
Line 10 in `post_info.html` there's a dublicate `- {% endif %}` from the previous unwanted change to remove.

Reply:
Sorry.